tools/downloader/src/open_model_zoo/model_tools/_configuration.py

```diff
@@ -359,9 +360,23 @@ def deserialize(cls, model, name, subdirectory):
             task_type = validate_string_enum('"task_type"', model['task_type'],
                 _common.KNOWN_TASK_TYPES)
 
+            composite_model_name = model.get('composite_model_name', None)
+            if composite_model_name is not None:
+                composite_model_name = validate_string('"composite_model_name"', composite_model_name)
```

Comment:
Any error message produced here would be incorrect - `model.yml` files do not _have_ `composite_model_name` keys.

And more broadly, it makes no sense to first add `composite_model_name` to `model` and then extract it back out. It's redundant, and it will produce strange results if someone does add a `composite_model_name` key to a `model.yml` file. Instead, you should just pass `composite_model_name` into this function as a parameter. Moreover, any validation on composite model names should happen when you first find the `composite-model.yml` file, not here.

Reply:
Done